openfisca_us/tests/policy/baseline/usda/snap/integration.yaml

```diff
@@ -18,13 +18,30 @@
     households:
       household:
         state_code_str: CA
-        state_group: CONTIGUOUS_US
   output:
-    snap: 3_051.6 # 8160 - 5108.4.
-    snap_max_benefit: 8_160 # 12 * $680 (parameter for family of 4 in contiguous US).
+    snap: 4_911.6 # 10_020 - 5108.4.
+    snap_max_benefit: 10_020 # 12 * $835 (parameter for family of 4 in contiguous US).
     snap_expected_contribution: 5_108.4 # 17028 * 0.3 (flat parameter).
     snap_net_income: 17_028 # 17028 - 0.
     snap_net_income_pre_shelter: 17_028 # 24000 - 2172 - 4800.
-    snap_shelter_deduction: 0 # No shelter costs reported.
+    snap_excess_shelter_expense_deduction: 0 # No shelter costs reported.
     snap_standard_deduction: 2_172 # 181 (parameter for family of 4 in contiguous US) * 12.
-    snap_earnings_deduction: 4_800 # 2000 * 0.2 (flat 20% parameter) * 12.
+    snap_earned_income_deduction: 4_800 # 2000 * 0.2 (flat 20% parameter) * 12.
+
+- name: Single person in CA at 200% of FPL gets minimum benefit via BBCE.
```

Comment:
@nikhilwoodruff this and the below test check the values for the `vary` issue we were chatting about. They pass as expected.

Reply:
Thanks for identifying the issue as `np.any` in `meets_snap_categorical_eligibility` needs an `axis=0` arg. Fixed now.